packages/labs/test/multiSelectTests.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { Classes, MenuItem } from "@blueprintjs/core";
+import { assert } from "chai";
+import * as classNames from "classnames";
+import { mount } from "enzyme";
+import * as React from "react";
+
+import { Film, TOP_100_FILMS } from "../examples/data";
+import { IMultiSelectProps, ISelectItemRendererProps, MultiSelect } from "../src/index";
+
+describe("<MultiSelect>", () => {
```

Comment:
Well, `placeholder` itself works as expected even without this fix! It's only when you both have a placeholder and the `onItemSelect` callback fires that this problem arises. I wrote a test for just checking that the `placeholder` propagates first but it wasn't really doing much :) I can add it back if you think it helps in terms of test completeness.